lib/binding_rust/lib.rs

```diff
@@ -93,6 +94,8 @@ pub enum LogType {
     Lex,
 }
 
+type FieldId = Option<NonZeroU16>;
```

Comment:
Oh, also, I think the `FieldId` should be *inside* of the `Option`, so the type of `field_id_for_name` would be `Option<FieldId>`.